versie 5.8.0-rc8 draait niet op Java 8

### Version
Which version of flamingo do you use: <!-- go to http://<server>/viewer-admin/about.jsp or http://<server>/viewer/about.jsp and paste below -->
5.8.0-rc8 (maar ook met  5.8.0-rc6)

### Server log
<details><summary>relevant section of the server log:</summary>

```

Error:  Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.8.1:compile (default-compile) on project framework: Compilation failure
Error:  /home/runner/work/OIGS/OIGS/framework/src/main/java/nl/b3p/oigs/framework/utils/Util.java:[28,19] cannot access nl.b3p.mail.Mailer
Error:    bad class file: /home/runner/.m2/repository/org/flamingo-mc/viewer-commons/5.8.0-rc8/viewer-commons-5.8.0-rc8.jar(nl/b3p/mail/Mailer.class)
Error:      class file has wrong version 55.0, should be 52.0
Error:      Please remove or make sure it appears in the correct subdirectory of the classpath.
Error:  -> [Help 1]
org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.8.1:compile (default-compile) on project framework: Compilation failure
/home/runner/work/OIGS/OIGS/framework/src/main/java/nl/b3p/oigs/framework/utils/Util.java:[28,19] cannot access nl.b3p.mail.Mailer
  bad class file: /home/runner/.m2/repository/org/flamingo-mc/viewer-commons/5.8.0-rc8/viewer-commons-5.8.0-rc8.jar(nl/b3p/mail/Mailer.class)
    class file has wrong version 55.0, should be 52.0
    Please remove or make sure it appears in the correct subdirectory of the classpath.

    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:215)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:156)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:148)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:117)
    at org.apache.maven.lifecycle.internal.builder.multithreaded.MultiThreadedBuilder$1.call (MultiThreadedBuilder.java:190)
    at org.apache.maven.lifecycle.internal.builder.multithreaded.MultiThreadedBuilder$1.call (MultiThreadedBuilder.java:186)
    at java.util.concurrent.FutureTask.run (FutureTask.java:266)
    at java.util.concurrent.Executors$RunnableAdapter.call (Executors.java:511)
    at java.util.concurrent.FutureTask.run (FutureTask.java:266)
    at java.util.concurrent.ThreadPoolExecutor.runWorker (ThreadPoolExecutor.java:1149)
    at java.util.concurrent.ThreadPoolExecutor$Worker.run (ThreadPoolExecutor.java:624)
    at java.lang.Thread.run (Thread.java:748)
Caused by: org.apache.maven.plugin.compiler.CompilationFailureException: Compilation failure
/home/runner/work/OIGS/OIGS/framework/src/main/java/nl/b3p/oigs/framework/utils/Util.java:[28,19] cannot access nl.b3p.mail.Mailer
  bad class file: /home/runner/.m2/repository/org/flamingo-mc/viewer-commons/5.8.0-rc8/viewer-commons-5.8.0-rc8.jar(nl/b3p/mail/Mailer.class)
    class file has wrong version 55.0, should be 52.0
    Please remove or make sure it appears in the correct subdirectory of the classpath.

```

</details>


### What did you expect?
<!-- What should have happened? --> het moet mogelijk zijn om v5.8.x libraries in een Java 8 project te gebruiken

--- a/.github/workflows/ubuntu-maven.yml
+++ b/.github/workflows/ubuntu-maven.yml
@@ -59,6 +59,8 @@ jobs:
           mvn -e clean test -B
           mvn -e verify -B -Pgh-action -pl '!viewer-admin'
           mvn -e verify -B -Pgh-action -pl 'viewer-admin'
+      - name: Verify Java classfile version
+        run: .jenkins/check-java-binary-version.sh
       - name: build javadoc
         run: |
           mvn javadoc:javadoc

--- a/.jenkins/check-java-binary-version.sh
+++ b/.jenkins/check-java-binary-version.sh
@@ -1,29 +1,16 @@
 #!/usr/bin/env bash
 # check of class file is java 8 == 52
-set -eE
 set _exitCode=0
 
-# van iedere module 1 classfile testen
-for clazz in 'form/target/classes/ nl.b3p.gbi.converter.Formulier' \
-            'solr-commons/target/classes/ nl.b3p.viewer.solr.SolrUpdateJob' \
-            'viewer/target/classes/ nl.b3p.viewer.util.FlamingoCQL' \
-            'viewer-admin/target/classes/ nl.b3p.viewer.admin.stripes.ApplicationActionBean' \
-            'viewer-audit/target/classes/ nl.b3p.viewer.audit.LoggingServiceFactory' \
-            'viewer-commons/target/classes/ nl.b3p.i18n.LocalizableActionBean' \
-            'viewer-config-persistence/target/classes/ nl.b3p.viewer.config.security.User' \
-            'web-commons/target/classes/ nl.b3p.web.stripes.ErrorMessageResolution'
+echo Controleer of alle .class bestanden Java 8 versie zijn...
 
-do
-  testvalues=($clazz)
-  printf "\nControleer classfile versie van ${testvalues[1]} in ${testvalues[0]}."
-  if [ $(javap -verbose -cp ${testvalues[0]} ${testvalues[1]} | grep -c 'major version: 52') == 1 ]; then
-    printf "\nOK: ${testvalues[1]} is een Java 8 classfile.\n"
-  else
-    printf "\nFAILED: ${testvalues[1]} is geen Java 8 classfile.\n"
-    _exitCode=1
-    _msg=' niet'
-  fi
-done
+while IFS= read -r -d '' clazz; do
+	file "$clazz" | grep -v 'version 52.0'
+	if [[ $? == 0 ]]; then
+		_exitCode=1
+		_msg=' niet'
+	fi
+done < <(find . -name "*.class" -print0)
 
 printf "\nControle$_msg succesvol afgerond.\n\n"
 exit $_exitCode

--- a/.jenkins/check-java-binary-version.sh
+++ b/.jenkins/check-java-binary-version.sh
@@ -1,10 +1,29 @@
 #!/usr/bin/env bash
 # check of class file is java 8 == 52
 set -eE
-if [ $(javap -verbose -cp viewer-audit/target/classes/ nl.b3p.viewer.audit.AuditMessageObject | grep -c 'major version: 52') == 1 ]; then
-  printf "\nOK: Java 8 classfile\n\n"
-  exit 0
-else
-  printf "\nFAILED: Geen Java 8 classfile\n\n"
-  exit 1
-fi
+set _exitCode=0
+
+# van iedere module 1 classfile testen
+for clazz in 'form/target/classes/ nl.b3p.gbi.converter.Formulier' \
+            'solr-commons/target/classes/ nl.b3p.viewer.solr.SolrUpdateJob' \
+            'viewer/target/classes/ nl.b3p.viewer.util.FlamingoCQL' \
+            'viewer-admin/target/classes/ nl.b3p.viewer.admin.stripes.ApplicationActionBean' \
+            'viewer-audit/target/classes/ nl.b3p.viewer.audit.LoggingServiceFactory' \
+            'viewer-commons/target/classes/ nl.b3p.i18n.LocalizableActionBean' \
+            'viewer-config-persistence/target/classes/ nl.b3p.viewer.config.security.User' \
+            'web-commons/target/classes/ nl.b3p.web.stripes.ErrorMessageResolution'
+
+do
+  testvalues=($clazz)
+  printf "\nControleer classfile versie van ${testvalues[1]} in ${testvalues[0]}."
+  if [ $(javap -verbose -cp ${testvalues[0]} ${testvalues[1]} | grep -c 'major version: 52') == 1 ]; then
+    printf "\nOK: ${testvalues[1]} is een Java 8 classfile.\n"
+  else
+    printf "\nFAILED: ${testvalues[1]} is geen Java 8 classfile.\n"
+    _exitCode=1
+    _msg=' niet'
+  fi
+done
+
+printf "\nControle$_msg succesvol afgerond.\n\n"
+exit $_exitCode

--- a/.jenkins/check-java-binary-version.sh
+++ b/.jenkins/check-java-binary-version.sh
@@ -1,16 +1,29 @@
 #!/usr/bin/env bash
 # check of class file is java 8 == 52
+set -eE
 set _exitCode=0
 
-echo Controleer of alle .class bestanden Java 8 versie zijn...
+# van iedere module 1 classfile testen
+for clazz in 'form/target/classes/ nl.b3p.gbi.converter.Formulier' \
+            'solr-commons/target/classes/ nl.b3p.viewer.solr.SolrUpdateJob' \
+            'viewer/target/classes/ nl.b3p.viewer.util.FlamingoCQL' \
+            'viewer-admin/target/classes/ nl.b3p.viewer.admin.stripes.ApplicationActionBean' \
+            'viewer-audit/target/classes/ nl.b3p.viewer.audit.LoggingServiceFactory' \
+            'viewer-commons/target/classes/ nl.b3p.i18n.LocalizableActionBean' \
+            'viewer-config-persistence/target/classes/ nl.b3p.viewer.config.security.User' \
+            'web-commons/target/classes/ nl.b3p.web.stripes.ErrorMessageResolution'
 
-while IFS= read -r -d '' clazz; do
-	file "$clazz" | grep -v 'version 52.0'
-	if [[ $? == 0 ]]; then
-		_exitCode=1
-		_msg=' niet'
-	fi
-done < <(find . -name "*.class" -print0)
+do
+  testvalues=($clazz)
+  printf "\nControleer classfile versie van ${testvalues[1]} in ${testvalues[0]}."
+  if [ $(javap -verbose -cp ${testvalues[0]} ${testvalues[1]} | grep -c 'major version: 52') == 1 ]; then
+    printf "\nOK: ${testvalues[1]} is een Java 8 classfile.\n"
+  else
+    printf "\nFAILED: ${testvalues[1]} is geen Java 8 classfile.\n"
+    _exitCode=1
+    _msg=' niet'
+  fi
+done
 
 printf "\nControle$_msg succesvol afgerond.\n\n"
 exit $_exitCode

--- a/Jenkinsfile
+++ b/Jenkinsfile
@@ -71,7 +71,7 @@ timestamps {
                 }
 
                 stage("check Java runtime version") {
-                    sh " .jenkins/check-java-binary-version.sh"
+                    sh ".jenkins/check-java-binary-version.sh"
                 }
                 
                 if (jdkTestName == 'OpenJDK11') {

